pkg/client_data/test/ensure_build_test.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// TODO(kevmoo): replace with a common utility
+//               https://github.com/dart-lang/build/issues/716
+@Tags(['presubmit-only'])
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:test/test.dart';
+
+final _pub = p.join(p.dirname(Platform.resolvedExecutable), 'pub');
+
+void main() {
+  test('ensure local build succeeds with no changes', () {
```

Comment:
Why is this not using `build_verify`?

Reply:
This is the only test in `test/` and if we don't have anything else, `pub run test` will complain: `No test files were passed and the default "test/" directory doesn't exist.`.

Reply:
But isn't exact what `build_verify` does?

https://pub.dev/packages/build_verify

Given that the package contains something that should be build, using this seems to make sense...

Reply:
It is, and you are right, but then we'd need to introduce fake tests just to run `pub run test`.